static/release-notes-hub.html

```diff
@@ -12,7 +12,9 @@
             font-style: normal;
             font-weight: normal;
             src: url("https://cdn.vsassets.io/v/M156_20190815.3/_ext/ms.vss-web/platform-content/fonts/Bowtie.eot?");
-            src: url("https://cdn.vsassets.io/v/M156_20190815.3/_ext/ms.vss-web/platform-content/fonts/Bowtie.eot?iefix") format("embedded-opentype"), url("https://cdn.vsassets.io/v/M156_20190815.3/_ext/ms.vss-web/platform-content/fonts/bowtie.woff") format("woff"), url("https://cdn.vsassets.io/v/M156_20190815.3/_ext/ms.vss-web/platform-content/fonts/Bowtie.svg#bowtie") format("svg")
+            src: url("https://cdn.vsassets.io/v/M156_20190815.3/_ext/ms.vss-web/platform-content/fonts/Bowtie.eot?iefix") format("embedded-opentype"),
```

Comment:
this url may be dynamic and should seek the way on how to get that url identifier into the package. Or else, maybe even add styles programatically during intialization.
Maybe check other open source extensions to see how they deal with fonts.

Reply:
But a fluentui theming, if works, may be better)